client/pom.xml

```
@@ -166,5 +166,15 @@
             <artifactId>oshi-core</artifactId>
             <version>5.3.2</version>
         </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
```

This looks like a typo, as you have the same dependency just a few lines down. I think you meant `micrometer-core`.

```suggestion
            <artifactId>micrometer-core</artifactId>
```